Stockwatch/Stockwatch.WindowsApp/StockSymbolPage.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Stockwatch.Business;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stockwatch.WindowsApp
+{
+    public interface IStockSymbolPage
+    {
+        void AddSymbol(string symbol);
+        List<string> GetSymbolList();
+    }
```

Comment:
Add a new line between entity defintions

Reply:
Now only 1 entity defintion. Have left line after that.